db/base/collection_keys.go

```diff
@@ -17,18 +17,18 @@ import (
 	"github.com/sourcenetwork/defradb/core"
 )
 
-// MakeIndexPrefix generates a key prefix for the given collection/index descriptions
-func MakeCollectionKey(col client.CollectionDescription) core.DataStoreKey {
+// MakeDSKeyWithCollectionID generates a key prefix for the given collection/index descriptions
```

Comment:
todo: Whilst the old name of this function seems odd, the new one doesn't seem very nice either.  I really dislike `DSKey` (instead of `DataStoreKey`).  And `DataStoreKey` itself is out of date and should probably be changed at somepoint (out of scope), meaning adding it to the function name just creates additional places to change when that type rename does happen.

If you really want to include the type name in the constructor name, can you please not abbreviate it, and instead use `DataStoreKey`?

Same change should be applied to the function below too.

Reply:
Sure so as pointed out in above discussion can do `MakeDataStoreKeyWithCollectionID`:

- [ ] rename: MakeDSKeyWithCollectionID to MakeDataStoreKeyWithCollectionID